bessctl/conf/testing/run_module_tests.bess

```diff
@@ -134,6 +135,7 @@ def load_test(filename):
         'CRASH_TEST_INPUTS': [],
         'OUTPUT_TEST_INPUTS': [],
         'CUSTOM_TEST_FUNCTIONS': [],
+        'protobuf_to_dict': pb_conv.protobuf_to_dict,
```

Comment:
Side note: this is no longer used, since the exact match test has been converted to the new framework. We could drop the changes to this file entirely now.